lib/helpers/MemberUiHelper.ts

```diff
@@ -0,0 +1,131 @@
+import {Page, Locator, expect} from "@playwright/test";
+import {UiBaseLocators} from "./UiBaseLocators";
+
+export class MemberUiHelper extends UiBaseLocators {
+  private readonly membersTab: Locator;
+  private readonly searchTxt: Locator;
+  private readonly memberNameTxt: Locator;
+  private readonly commentsTxt: Locator;
+  private readonly memberTab: Locator;
+  private readonly detailsTab: Locator;
+  private readonly usernameTxt: Locator;
+  private readonly emailTxt: Locator;
+  private readonly passwordTxt: Locator;
+  private readonly confirmPasswordTxt: Locator;
+  private readonly newPasswordTxt: Locator;
+  private readonly approvedSlider: Locator;
+  private readonly lockedOutSlider: Locator;
+  private readonly twoFactorAuthenticationSlider: Locator;
+  private readonly memberInfoItems: Locator;
+  private readonly changePasswordBtn: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.membersTab = page.locator('uui-tab[label="Members"]');
+    this.searchTxt = page.locator('#input-search');
+    this.memberNameTxt = page.locator('#name-input #input');
+    this.commentsTxt = page.locator('#textarea');
+    this.memberTab = page.locator('uui-tab').filter({hasText: 'Member'}).locator('svg');
+    this.detailsTab = page.locator('uui-tab').filter({hasText: 'Details'}).locator('svg');
+    this.usernameTxt = page.locator('[label="Username"] #input');
+    this.emailTxt = page.locator('[label="Email"] #input');
+    this.passwordTxt = page.locator('[label="Password"] #input');
+    this.confirmPasswordTxt = page.locator('[label="Confirm password"] #input');
+    this.newPasswordTxt = page.locator('[label="New password"] #input');
+    this.approvedSlider = page.locator('[label="Approved"] #slider');
+    this.lockedOutSlider = page.locator('[label="Locked out"] #slider');
+    this.twoFactorAuthenticationSlider = page.locator('[label="Two-Factor authentication"] #slider');
+    this.memberInfoItems = page.locator('.general-item');
+    this.changePasswordBtn = page.getByLabel('Change password', {exact: true});
+  }
+
+  async clickMembersTab() {
+    await this.membersTab.click();
+    await this.page.waitForTimeout(200);
+  }
+
+  async clickMemberTab() {
+    await this.memberTab.click();
+    await this.page.waitForTimeout(200);
+  }
+
+  async clickDetailsTab() {
+    await this.detailsTab.click();
+    await this.page.waitForTimeout(200);
+  }
+
+  async clickMemberLinkByName(memberName: string) {
+    await this.page.getByRole('link', {name: memberName}).click();
+    await this.page.waitForTimeout(200);
+  }
```

Comment:
Is there a way we can avoid the wait for timeout? There should be some locators we can expect